error.go

```diff
@@ -4,27 +4,47 @@ import (
 	"strings"
 )
 
+// Error type returned when checking responses
+// that contain an error.
 type ResponseError struct {
+	// High level status of the response, api_schema.go
 	statusCode int
+	// Project specific error response, see project that uses it
+	errorCode int
+	// Human readable (but not beautified) error text
 	statusText string
```

Comment:
Do we want to have JSON tags here? In case of `statusCode` 200 we can leave `errCode` empty.